### PR TITLE
jwt basic token

### PR DESCRIPTION
**Pull request by festinalente91**
- to implement OAuth 2.0, I needed to generate a token
- to use on development environment you need to set two process.env variables.
- JWT_SECRET_KEY : secret key for encrypt and decrypt
- JWT_TOKEN_EXPIRE : token expired term

### CODE REVIEW

File: src/server/auth-token.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+/*
+  @TODO
+  for better security, you need to encrypt payload(claims in jwt)
+ */
+class AuthToken {
+
+  constructor() {}
+
+  encode(payload) {
```
Comment (hewonjeong) [inline src/server/auth-token.js:15]: Well done~!

File: src/server/auth-token.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        if (err) {
+          return reject(err);
+        }
+        return resolve(payload);
+      })
+    });
+  }
+}
+
+export default AuthToken = new AuthToken(SECRET_KEY);
```
Comment (ovekyc) [inline src/server/auth-token.js]: There is only empty constructor

Comment (festinalente91) [reply src/server/auth-token.js]: you are right !

**Comment by hewonjeong** [comment]
LGTM

**Comment by ovekyc** [comment]
:LGTM

**Comment by festinalente91** [comment]
LGTM nothing has changed

**Pull request merged**